internal/web_package/web_package.bzl

```diff
@@ -15,6 +45,18 @@ def html_asset_inject(index_html, action_factory, injector, rootDirs, assets, ou
     return output
 
 def move_files(output_name, files, action_factory, assembler, root_paths):
+    """Moves files into an output directory
+
+    Args:
+      output_name: the name of the output directory
+      files: the files to move
+      action_factory: TODO
```

Comment:
@alexeagle Can you comment with best descriptions for these?

Reply:
same - it's `ctx.actions`